src/main/java/au/gov/notify/NotificationClientApi.java

```diff
@@ -1,7 +1,13 @@
-package uk.gov.service.notify;
+package au.gov.notify;
+
+import au.gov.notify.dtos.Notification;
```

Comment:
What's the reason for moving these classes to dtos? Just wondering.

Reply:
It was to move all request/response classes into their own folder to provide more structure  rather than having everything in a single folder/package. Personal preference.